book/website/community-handbook/coworking/coworking-dropins.md

```diff
@@ -0,0 +1,90 @@
+The schedule also allows interested attendees to reference when to attend, based on the individual or team that is hosting on a particular week.
+
+In order to sign up, an interested attendee will directly message the hosts for that session on the organisation's collaboration platforms, such as Slack, Microsoft Teams, or e-mail.
+The hosts will then assign the attendee a 30-60 minute slot.
+
+## Resources for drop-in sessions
+
+The resources used for these sessions help facilitate coordination amongst the organisers and also to enable attendee sign up.
+
+1. Online call by [Zoom](https://www.zoom.us/): The Zoom link is shared on an internal Turing webpage and on Slack.
+2. In person location (optional): A reserved meeting room or location at a shared organisation space
+3. Online github repo: share information about the session, including [the schedule](https://github.com/alan-turing-institute/infrastructure-drop-ins/wiki/Schedule) and [how to sign up](https://github.com/alan-turing-institute/infrastructure-drop-ins)
+4. Engagement and impact tracking resource: Sheet with information on the topics raised, next steps, and the overall impact of the session
+
+These sessions are not recorded. 
+Breakout rooms may be created if there are multiple sessions happening at the same time.
+In that case, one host will stay in the main room to welcome new attendees and other hosts can meet with attendees in different breakout rooms.
+
+## Format of the session
+
+This format is drawn from the [Turing Infrastructure Drop-in Sessions](https://github.com/alan-turing-institute/infrastructure-drop-ins) which uses the following format:
+
+| Duration | Activity |
+| ---- | -------- |
+| 5 mins | 👋 Welcome attendees and assign slot |
+| 30 mins | :speech_balloon: Slot 1 |
+| 30 mins | :speech_balloon: Slot 2 |
+| 30 mins | :speech_balloon: Slot 3 |
+| 30 mins | :speech_balloon: Slot 4 |
+| 5 mins | 👋 Close session |
+
+Each slot may run longer or shorter, but the standard length is 30 minutes.
+Multiple slots may run in parallel through breakout rooms.
+
+## Chairing drop-in sessions
+
+### Before the Session
+
+- Session hosts can double check the schedule to make sure the correct information is documented for that session
+    - Update host names and affiliations
+    - Correct Zoom link and physical location of the session
+    - Share session on Slack and other channels inviting attendees to the event
+- Session hosts can coordinate amongst each other to schedule and assign slots during the session
+
+#### Approaches to Sign ups
+
+There are many ways to manage sign ups that occur before the call and during the call.
+The best method will depend on the size of your team.
+Sign ups are a way to approach scheduling different attendee requests
+Here we share a few approaches, ranging from low touch to high touch:
+- Attendees drop in at any point during a session. This may require them to sit on other conversations and wait to have their question answered.
```

Comment:
```suggestion
- Attendees drop in at any point during a session. This may require them to sit in on other conversations and wait to have their question answered.
```